README.md

```diff
@@ -95,16 +95,27 @@ cargo run --bin spacewalk-standalone --release -- --dev
 
 # Development
 
-To ease the maintenance of proper code formatting, please consider installing the pre-commit hook contained in this
+To ease the maintenance of proper code formatting and good practices, please consider installing the pre-commit hook and/or the pre-push hook contained in this
 repository.
-This hook runs `rustfmt` on all the changed files in a commit, to ensure that changed files are properly formatted
+
+The pre-commit hook runs `rustfmt` on all the changed files in a commit, to ensure that changed files are properly formatted
 before committing.
 You can install the hook by running the following commands.
 
 ```
 .maintain/add-pre-commit.sh
 ```
 
+The pre-push hool runs clippy checks that are also performed in the CI of the repository. These checks need to be successful so the push actually happens.
+Otherwise, pelase run the corresponding clippy fix command or manually fix the issue.
+
+To install the hook, run:
+```
+.maintain/add-prepush-hook.sh
```

Comment:
I tested this locally and when I wanted to push, I got the message
```
hint: The '.git/hooks/pre-push' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
```

so I made it executable with `chmod u+x .git/hooks/pre-push`. Maybe we should mention this here as well?

Reply:
> Did you have to do the same with the pre-commit one right? 

I don't think so, was also confused that I had to do it this time. But the script in `.maintain` itself was working fine, I just had to make the generated script executable. Maybe it's different between macOS and linux? Not sure where you tested it.

Reply:
Apparently it was needed in macOS but not on linux!